docs/run_test/quick_run.rst

```diff
@@ -59,7 +59,10 @@ Linux VM on Azure.
    attribute ``isDefault`` as true, it's not necessary to do so as long
    as you provide the correct ``<subscription id>``.
 
-#. Prepare SSH key pair
+#. Prepare SSH key pair (Optional)
```

Comment:
Please move this section into run in different platform to introduce the parameter. The quick start should be as quick as possible.